src/System.Security.Cryptography.X509Certificates/tests/LoadFromFileTests.cs

```diff
@@ -167,7 +167,7 @@ public static void TestLoadSignedFile()
                 byte[] expectedThumbprint = expectedThumbprintHash.HexToByteArray();
                 byte[] actualThumbprint = c.GetCertHash();
                 Assert.Equal(expectedThumbprint, actualThumbprint);
-#if netstandard17
+#if netcoreapp11
```

Comment:
Ugh. Reverted those. However I don't think we are differentiating between the two across other code (at least consistently)